ajax/libs/jquery-jsonview/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "jquery-jsonview",
+  "version": "1.2.3",
+  "description": "jQuery plugin for display pretty JSON.",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/yesmeck/jquery-jsonview.git"
+  },
+  "keywords": [
+    "json"
+  ],
+  "author": "Meck <yesmeck@gmail.com>",
+  "license": "MIT",
+  "homepage": "https://github.com/yesmeck/jquery-jsonview",
+  "npmName": "jquery-jsonview",
+  "npmFileMap": [
+    {
+      "basePath": "/dist",
+      "files": [
+        "**/*"
```

Comment:
I think it should be `"**/*.+(css|js)"` or `"**/*.+(css|js|map)"`, there is a html file in `dist`